windows/RNSVG/TextView.cpp

```diff
@@ -45,17 +46,20 @@ void TextView::UpdateProperties(IJSValueReader const &reader, bool forceUpdate,
   __super::UpdateProperties(reader, forceUpdate, invalidate);
 }
 
-void TextView::RenderGroup(UI::Xaml::CanvasControl const &canvas, CanvasDrawingSession const &session) {
-  auto const &transform{session.Transform()};
+void TextView::DrawGroup(RNSVG::D2DDeviceContext const &context, Size const &size) {
+  com_ptr<ID2D1DeviceContext> deviceContext{get_self<D2DDeviceContext>(context)->Get()};
+
+  D2D1_MATRIX_3X2_F transform{D2DHelpers::GetTransform(deviceContext.get())};
+
   bool translateXY{X().Size() > 0 || Y().Size() > 0};
   if (translateXY) {
     float x{X().Size() > 0 ? X().GetAt(0).Value() : 0};
     float y{Y().Size() > 0 ? Y().GetAt(0).Value() : 0};
-    session.Transform(transform * Numerics::make_float3x2_translation(x, y));
+    deviceContext->SetTransform(D2D1::Matrix3x2F::Translation({x,y}) * transform);
```

Comment:
Isn't matrix math not commutative? Shouldn't this be:

```suggestion
    deviceContext->SetTransform(transform * D2D1::Matrix3x2F::Translation({x,y}));
```

Reply:
Correct that matrix multiplication is not commutative, but this is the correct order.